api/base/serializers.py

```diff
@@ -1837,3 +1839,27 @@ class MaintenanceStateSerializer(ser.ModelSerializer):
     class Meta:
         model = MaintenanceState
         fields = ('level', 'message', 'start', 'end')
+
+
+class HideIfSwitch(ConditionalField):
```

Comment:
Why have this if it's only for the DisableIfSwitch to subclass?

Reply:
You might want to hide a field that you can modify via the API sometime in the future, it's a nice QA tool to test setting with alarming users with new visible data.